docs/manuals/gui/pages.md

```diff
@@ -232,6 +232,8 @@ for all its pages.
     If you navigate to '/page2', the main title remains on the page, and the sub-title
     is replaced by the text 'This is page 2'
 
+    !!! info "For running Taipy Gui service with the other Taipy services, please refer to page [Running Taipy services](../running_services/index.md)"
```

Comment:
"For running Taipy Gui service"
->
"To run the Taipy GUI service"

"to page [Running Taipy services](https://github.com/Avaiga/taipy-doc/pull/running_services/index.md)"
to
"to the [Running Taipy services](https://github.com/Avaiga/taipy-doc/pull/running_services/index.md) page"